lib/src/progress_bar.dart

```diff
@@ -57,7 +57,7 @@ class _VideoProgressBarState extends State<VideoProgressBar> {
     super.deactivate();
   }
 
-  Duration _calcRelativePosition(Offset globalPosition) {
+  Duration _calcRelativePosition(BuildContext context, Offset globalPosition) {
```

Comment:
This should be an extension method on `BuildContext`.

What that means is that this method should be an extension method of that class.

```dart
extension RelativePositionExtensions on BuildContext {
  Duration calcRelativePosition(Duration videoDuration, Offset globalPosition) {
    final box = this.findRenderObject()! as RenderBox;
    final Offset tapPos = box.globalToLocal(globalPosition);
    final double relative = tapPos.dx / box.size.width;
    final Duration position = videoDuration * relative;
    return position;
  }
}
```

Then, whenever you need to use it, simply call `context.calcRelativePosition(controller.value.duration, globalPosition)`.